mne_bids/read.py

```diff
@@ -296,6 +355,36 @@ def read_raw_bids(bids_fname, bids_root, extra_params=None,
     if channels_fname is not None:
         raw = _handle_channels_reading(channels_fname, bids_fname, raw)
 
+    # Try to find an associated channels.tsv to get information about the
+    # status and type of present channels
+    electrodes_fname = _find_matching_sidecar(bids_fname, bids_root,
+                                              'electrodes.tsv',
+                                              allow_fail=True)
+    coordsystem_fname = _find_matching_sidecar(bids_fname, bids_root,
+                                               'coordsystem.json',
+                                               allow_fail=True)
+    if electrodes_fname is not None:
+        if coordsystem_fname is None:
+            raise RuntimeError("BIDS mandates that the coordsystem.json "
+                               "should exist if electrodes.tsv does. "
+                               "Please create coordsystem.json for"
+                               "{}".format(bids_basename))
+        # Get MRI landmarks from the JSON sidecar
+        with open(coordsystem_fname, 'r') as fin:
+            coordsystem_json = json.load(fin)
+
+        if kind == "meg":
+            coord_frame = coordsystem_json['MEGCoordinateSystem']
+        elif kind in ["ieeg", "seeg", "ecog"]:
```

Comment:
kind can be seeg or ecog?

Reply:
@adam2392 can you clarify this?

Reply:
Ah sorry about that. I just up some additional in-line comments and addressed this.